app/services/transient_registration_cleanup_service.rb

```diff
@@ -2,8 +2,21 @@
 
 class TransientRegistrationCleanupService < WasteCarriersEngine::BaseService
   def run
-    transient_registrations_to_remove.destroy_all
-    no_created_at_transient_registrations_to_remove.destroy_all
+    transient_registrations_with_created_at = transient_registrations_to_remove
+    transient_registrations_without_created_at = no_created_at_transient_registrations_to_remove
+
+    # rubocop:disable Rails/Output
+    # :nocov:
+    unless Rails.env.test?
+      puts "Removing transient_registrations created up to #{oldest_possible_date} excluding #{excluded_states}"
+      puts "Removing #{transient_registrations_with_created_at} transient_registrations with a created_at value"
```

Comment:
Will this display a number? Looks to me like it will display a bunch of objects?

Reply:
True - fixed